crates/core_simd/src/swizzle.rs

```diff
@@ -451,6 +495,24 @@ where
         unsafe { Self::from_int_unchecked(self.to_int().rotate_elements_right::<OFFSET>()) }
     }
 
+    /// Shifts the mask elements to the left by `OFFSET`, filling in with
+    /// `padding` from the right.
+    #[inline]
+    #[must_use = "method returns a new vector and does not mutate the original inputs"]
+    pub fn shift_elements_left<const OFFSET: usize>(self, padding: T) -> Self {
```

Comment:
I think `padding` should be `bool` for masks

Reply:
I added this change but the conversion felt awkward. LMK if there is a more idiomatic way.

Reply:
I think that's the way to go